src/frontend/magnify/src/components/design-system/Layout/Header/Avatar/ResponsiveLayoutHeaderAvatar.test.tsx

```diff
@@ -1,15 +1,19 @@
-import { render, screen } from '@testing-library/react';
+import { fireEvent, render, screen } from '@testing-library/react';
 import React from 'react';
-import { BrowserRouter } from 'react-router-dom';
+import { commonTranslation } from '../../../../../i18n/Messages/commonMessages';
+import { MagnifyTestingProvider } from '../../../../app';
 import { ResponsiveLayoutHeaderAvatar } from './ResponsiveLayoutHeaderAvatar';
 
 describe('Should show a ResponsiveLayoutHeaderAvatar', () => {
```

Comment:
we name the test file likes `Component.spec.tsx` in richie, maybe ask @jbpenrath what he thinks

Reply:
That's true in other projects, we use `.spec.tsx` suffix. That is important from my point of view is to always use the same suffix. In this project this is `.test.tsx` that's ok for me.

Reply:
I would say uniformity across projects is also important.

Reply:
Do this in another PR